docs/rule/no-jsx-attributes.md

```diff
@@ -0,0 +1,66 @@
+# no-jsx-attributes
+
+Folks coming from React may have developed habits around how they type attributes on elements.
+JSX isn't HTML (it's JS), so in JS, you can't have kebab-case identifiers, so JSX uses camelCase.
+
+However, since Ember uses HTML, camelCase attributes are not valid when writing components.
+
+## Examples
+
+This rule **forbids** the following attributes:
+
+- acceptCharset
```

Comment:
should we also specify correct name here?
`acceptCharset` -> `{{property 'acceptCharset'}}`